src/main.cpp

```diff
@@ -101,6 +101,10 @@ int main(int argc, const char **argv) {
 	parser->define("qj", &options->jointQuantizationBits)
 		->description("joint indices and weights quantization bits used in Draco compression extension");
 
+	parser->define("uint", &options->useUintIndices)
+		->defaults(false)
+		->description("use OES_element_index_uint extension to allow 32-bit vertex indices");
```

Comment:
Maybe change this description to something like "Write indices as unsigned 32-bit integers" and change the option to `useUint32Indices` for clarity. The idea of using `OES_element_index_uint` is abstracted out to the person writing the glTF loader and may be confusing for people who are just looking to convert models.